src/market.py

```diff
@@ -10,7 +11,15 @@
 from ._layouts.account_flags import ACCOUNT_FLAGS_LAYOUT
 from ._layouts.market import MARKET_LAYOUT, MINT_LAYOUT
 from ._layouts.slab import Slab
+from .enums import Side
 from .instructions import DEFAULT_DEX_PROGRAM_ID
+from .queue_ import decode_event_queue, decode_request_queue
```

Comment:
I think we need to name it to `_queue.py` to hide it.

Reply:
I used `.queue_` to avoid a similar package that is called `queue` that is used in the import tools. If I call it `queue` it will report an error saying: `ImportError: attempted relative import with no known parent package`